modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties

```
@@ -1330,6 +1330,7 @@ stock-quantity-x=Stock Quantity: {0}
 store-currency=Store Currency
 store-management=Store Management
 subject-to-vat=Subject to VAT
+submitted-by=Submitted by
```

@riccardo-alberti  we should capitalize `by` since its a title.
```suggestion
submitted-by=Submitted By
```